deploy/helm/templates/_helpers.tpl

```diff
@@ -42,6 +42,15 @@ Usages:
 {{- printf "%s%s" $result $suffix -}}
 {{- end -}}
 
+{{/*
+Get cluster UID based on name and uid provided in .Values.cluster.
+Usage:
+  {{ include "common.cluster-uid" . }}
+*/}}
+{{- define "common.cluster-uid" -}}
```

Comment:
I put the UID logic into this template to make it easier to change it in the future. But right now, with a simple "default", it's not that much beneficial.